schemas/org.gnome.shell.extensions.clipboard-indicator.gschema.xml

```diff
@@ -77,6 +77,14 @@
         </description>
     </key>
 
+    <key name="move-item-first" type="b">
+      <default>true</default>
```

Comment:
Can you please change the default value to ``false``?
I would like this to be the default behavior.

Reply:
Thank you for reviewing. I've changed it to `false`.